master/usage/configuration/bgp.md

```diff
@@ -27,12 +27,17 @@ nodes to use the same AS Number.
 The full node-to-node mesh option provides a mechanism to automatically
 configure peering between all Calico nodes.  When enabled, each Calico node
 automatically sets up a BGP peering with every other Calico node in the
-network.  By default this is enabled.  However, the full node-to-node mesh is
-only useful for small scale deployments and where all Calico nodes are on the
-same L2 network.  We generally expect the full node-to-node mesh to be disabled
-and explicit BGP peering to be configured.
+network.  By default this is enabled.
 
-Explicit BGP peers may be configured globally or for a particular node.
+The full node-to-node mesh provides a simple mechanism for auto-configuring
+the BGP network in small scale deployments (say 50 nodes - although this limit
+is not set in stone and Calico has been deployed with beyond 100 nodes in a full
```

Comment:
Super minor nit.  `with beyond` doesn't flow well for me when I read it.  If it works well for you though then I'm good.